### PR TITLE
Create Self Growth-Optional not Mandatory 

### PR DESCRIPTION
**Pull request by ScarlettsVintage**
00. Field Work/ Experience 
01. Acknowledgement of Self eg. Where do you see yourself in 5 years? 10 years? Don’t share either! Based on that what do you have to do to get there ? Generic superficial plan.  02. Asses and Reasses your current short terms goals to long term goals. Every year or every other year if you don’t want to waste time.  03. Waste of time or efficiency ?
04. Teach someone to  eventually replace  you … start looking for someone who can help you grow.

### CODE REVIEW

**Comment by acd124** [comment]
This pull request does not seem to add anything, nor has it meaningfully presented foundations for why related changes would be necessary or desirable. Closing until such rationale or meaningful additions are presented. If you would like to start over with presenting rationale or a problem that needs solving, feel free to open an issue that may lead to a pull request in the future.

**Comment by ScarlettsVintage** [comment]
> 0. Field Work/ Experience
> 1. Acknowledgement of Self eg. Where do you see yourself in 5 years? 10 years? Don’t share either! Based on that what do you have to do to get there ? Generic superficial plan.  02. Asses and Reasses your current short terms goals to long term goals. Every year or every other year if you don’t want to waste time.  03. Waste of time or efficiency ?
> 2. Teach someone to  eventually replace  you … start looking for someone who can help you grow. And vise versa. 
> My sincere apologies if it does not serve you. Wrong timing!. Thank you for your concern. 

 No such thing as bad Feedback, all is acknowledged.

File: Fact Checking/Self Growth [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ScarlettsVintage) [inline Fact Checking/Self Growth:1]: [](https://marketplace.visualstudio.com/items?itemName=aiqubit.gemini-pro)

**Review by ScarlettsVintage — COMMENTED** [review]
@tandpfun 👻

**Pull request closed without merging**